tests/Routes/MailchimpRouteTest.php

```diff
@@ -25,8 +25,8 @@ protected function _before()
    */
   public function testRestCallSuccessfulWhenAddingNewMembers()
   {
-    $request = new \WP_REST_Request('GET', $this->route_endpoint->get_route_uri());
-    $request->params['GET'] = [
+    $request = new \WP_REST_Request('POST', $this->route_endpoint->get_route_uri());
```

Comment:
You could extract this `'POST'` string in a constant and then just changed it in one place, the constant would remain the same in other places but referring to the `'POST'` string.

Reply:
I need to be able to mock both GET and POST requests here (this is a mock WP_REST_Request). Tho I do agree since it's the same for all tests in this file, it could just be extracted into a const. 👍